### PR TITLE
Use integrated provider naming

### PR DESCRIPTION
**Pull request by lokst**
Updates our support articles to consistently adopt the "integrated provider" / "integrated DNS provider" / "integrated zone" terms

### CODE REVIEW

File: content/articles/integrated-dns-providers.markdown [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  - route53:GetHostedZone
+  - route53:DeleteHostedZone
+- Administrator access to a DNSimple account
+
+## Supported record types {#supported-record-types}
+
+Records of supported record types can be [synced](/articles/deployment-editor#record-syncing) from/to DNSimple to/from an integrated DNS provider.
+However, record fields not supported by DNSimple or the destination provider will be ignored. For example, record fields, like [record notes](/articles/record-notes) and [regional record](/articles/regional-records) information, are only supported in DNSimple records. They will not be synced to the integrated DNS provider.
+
+#### Supported Route 53 record types
```
Comment (kojdm) [inline content/articles/integrated-dns-providers.markdown:48]: I think we'll need to add URL here once https://github.com/dnsimple/dnsimple-app/pull/22728 is merged

**Review by kojdm — APPROVED** [review]
LGTM!

**Review by dallasread — APPROVED** [review]
Excellent! Thanks for doing this!

File: content/articles/integrated-dns-providers.markdown (hunk 1 above)
Comment (dallasread) [reply content/articles/integrated-dns-providers.markdown:48]: Perhaps with a note that explains a bit (eg. it is transformed into an A record).

Comment (lokst) [reply content/articles/integrated-dns-providers.markdown:48]: Yes, I'll open another PR to include the URL record type after https://github.com/dnsimple/dnsimple-app/pull/22728 is merged 👍

**Review by itsalyse — APPROVED** [review]

**Pull request merged**